assignments/assignment01.rb

```diff
@@ -38,12 +38,21 @@ def number_to_string(n, lang)
 # it accepts a string
 # and returns the same string with each word capitalized.
 def titleize(s)
-  words = s.split
-  caps = []
-  words.each do |word|
-    caps << word.capitalize
+
+  # first downcase the entire string
+  s.downcase!
```

Comment:
don't modify the input argument, so don't use the ! method. Instead, use the non-destructive version and assign the result to a local variable.